src/virtualenv/create/via_global_ref/builtin/pypy/common.py

```diff
@@ -44,9 +44,13 @@ def _add_shared_libs(cls, interpreter):
         # https://bitbucket.org/pypy/pypy/issue/1922/future-proofing-virtualenv
         python_dir = Path(interpreter.system_executable).resolve().parent
         for libname in cls._shared_libs():
-            src = python_dir / libname
-            if src.exists():
+            # Windows needs two dlls
+            found = False
+            for src in python_dir.glob(libname):
+                found = True
                 yield src
+            if not found:
+                raise RuntimeError(f"could not find {python_dir / libname}, aborting")
```

Comment:
We don't raise hard errors. Just because this creator doesn't to match does not mean that another plugin does not provide it, please remove this. If this is the case the creator should just say it cannot handle this interpreter :thinking: and should be checked in the can_create instead.